src/components/Footer.astro

```diff
@@ -8,21 +8,36 @@ const { styling } = Astro.props
   <div
     class={classNames(
       'container mx-auto flex w-full flex-col items-center justify-between gap-8 border-t px-5 py-14 md:flex-row md:gap-0 md:px-10 lg:px-12',
-      `${styling ? 'text-black' : 'text-white'} `,
+      `${styling ? 'text-black' : 'text-white'}`,
+      `${styling ? 'border-black' : 'border-white'}`,
     )}
   >
-    <a class="text-lg font-bold leading-7 tracking-wide md:text-xl lg:text-2xl lg:leading-7" href="mailto:aleksandra@alexuix.design"
-      >aleksandra@alexuix.design</a
+    <a
+      class={classNames(
+        'text-lg font-bold leading-7 tracking-wide md:text-xl lg:text-2xl lg:leading-7',
+        `${styling ? 'hover:opacity-80' : 'hover:text-secondary'}`,
+      )}
+      href="mailto:aleksandra@alexuix.design">aleksandra@alexuix.design</a
```

Comment:
This value is considered a magic number value. If you have some values that are constant and won't be changed in the foreseeable future you need to place them in a variable.

Example wise: `CONTACT_LINKEDIN_URL`, `CONTACT_EMAIL_ADDRESS` etc.

Also, ESLint failed the tests, please fix the errors.

In this particular case, you wouldn't have to type the string twice, you would use a `CONTACT_EMAIL_ADDRESS`.